PlatformConfigurations/IEEE802.1xPortBasedAuthentication.md

```diff
@@ -0,0 +1,872 @@
+  (MS-CHAPv2) or other methods.
+
+To configure the type of EAP to use, edit the `/etc/freeradius/3.0/mods-available/eap` file:
+
+Set the password of the private key in the `private_key_password` parameter:
+
+```
+eap {
+...
+  tls-config tls-common {
+...
+    private_key_password = key_password
+...
+  }
+}
+```
+
+For our configuration demo, we will use "whatever" from the default certificate configuraiton
+as the `key_password`:
+
+On the FreeRADIUS-Server:
+
+```
+eap {
+...
+  tls-config tls-common {
+...
+    private_key_password = whatever
+...
+  }
+}
+```
+
+Next, depending on your environment,
+set the `default_eap_type` parameter in the eap directive to the primary `EAP_type` you want to use:
+
+```
+eap {
+...
+  default_eap_type = EAP_type
+...
+}
+```
+
+For our configuration demo, we will use `ttls`:
+
+On the FreeRADIUS-Server:
+
+```
+eap {
+...
+  default_eap_type = ttls
+...
+}
+```
+
+To leave only EAP enabled for the outer authentication and disable plain-text authentication methods
+edit the `/etc/raddb/sites-available/default file`, and comment out all authentication methods other than eap.
+
+On the FreeRADIUS-Server:
+
+```
+authenticate {
+...
+    # Auth-Type PAP {
+    #     pap
+    # }
+
+    # Auth-Type CHAP {
+    #     chap
+    # }
+
+    # Auth-Type MS-CHAP {
+    #     mschap
+    # }
+
+    # mschap
+
+    # digest
+    ...
+}
+```
+
+Next, to specify if other hosts should be able to access the free RADIUS service, such as an Authenticator
+edit the `/etc/freeradius/3.0/clients.conf` file and use the following template:
+
+```
+client <hostname_or_description> {
+    ipaddr = <IP_address_or_range>
+    secret = <client_password>
+}
+```
+
+In our configuration, we will add the address of the bridge.
+
+On the FreeRADIUS-Server:
+
+```
+client DENT-Authenticator {
+        ipaddr          = 192.168.1.2
+        secret          = testing123
+}
+```
+
+If you plan to use EAP-TTLS or EAP-PEAP, be sure to also add the users to the `/etc/raddb/users file`:
+
+```
+example_user        Cleartext-Password := "user_password"
+```
+
+For our configuration demo, since we will use ttls, add the Supplicant's `identity` and `password`.
+
+On the FreeRADIUS-Server:
+
+```
+Supplicant       Cleartext-Password := "password"
+```
+
+For users who intend to use certificate-based authentication (EAP-TLS), do not add any entry.
+
+To verify the configuration files run `freeradius -XC`.
+
+On the FreeRADIUS-Server:
+
+```
+root@localhost:~# freeradius -XC
+FreeRADIUS Version 3.2.2
+Copyright (C) 1999-2022 The FreeRADIUS server project and contributors
+[...]
+Configuration appears to be OK
+```
+
+_NOTE: If port 1812 is said to already be in use, try restarting the freeradius service with
+`systemctl stop freeradius` then `freeradius -X`_
+
+### Configuring hostapd
+
+The host access point daemon (hostapd) service is useful tool
+for acting as an authenticator in a wired network. Upon configuration `hostapd_cli`
```

Comment:
It is not just a useful tool. Just say "we'll use hostapd as the authenticator".

Reply:
I have now reworded this section: 

![image](https://github.com/user-attachments/assets/9eaced1c-861f-4a7f-8dba-0e76fe118680)